### ISSUE TITLE
nintendo switch won't start after logo, code error 2153-1542

### ISSUE THREAD

**Issue by benweislo** [benweislo/tchap#1]
Hello everyone, my switch is blocked after the logo nintendo, with or without the cfw and atmosphere. i tried to use the backup with hekate but nothing works. 
![20191001_120354](https://user-images.githubusercontent.com/56071103/66036793-2ccead80-e51f-11e9-835f-724d44f14ac8.jpg)
The nintendo's firmware was 9.0.0 and atmosphere was 0.9.4.
i did nothing in particular, i installed games but when i switched back on i had this error..
Please can you help ? i had it just 2 days ago
thanks in advance ^^

**Comment by brodrrr** [comment]
If you solve the problem tell me I am looking for now to talk 901 apocalypse with me

**Comment by benweislo** [comment]
yep, but no, still not found the problem. I think it might come from the heat of the console itself from what i understood on other forums but no one could find the problem.
I could find a way to restart it and put it like it wasss empty/new. But as soon as i try to do anything with it, it goes back to that problem. usually it's the 2153-1540 first (after few minutes of playing) then the 2153-1542 witch never goes away... until i erase, restore blablabla and i could start it again.. I'm waiting for some info from the store i bought it from because it might come from the fan
